packages/@adobe/spectrum-css-temp/components/menu/index.css

```diff
@@ -218,6 +218,13 @@ governing permissions and limitations under the License.
   padding-block-start: var(--spectrum-global-dimension-size-10);
   padding-inline-end: var(--spectrum-global-dimension-size-100);
 }
+.spectrum-Menu-avatar {
+  /* Matches icon. */
+  grid-area: icon;
+  padding-block-start: var(--spectrum-global-dimension-size-10);
+  padding-inline-end: var(--spectrum-global-dimension-size-100);
+  overflow: visible;
```

Comment:
I think if we use margin instead of padding, we don't need this overflow visible line